cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	parrotOrgId          int
+	parrotPartitionCount int32
```

Comment:
these vars don't need the 'parrot' prefix

Reply:
They don't need it, but I felt like the `cmd/fakemetrics` namespace was getting a little crowded.  No longer a concern if we're moving to `cmd/mt-parrot`